requirements.txt

```
@@ -1,4 +1,5 @@
 qiskit>=0.44
+qiskit-algorithms
```

```suggestion
qiskit-algorithms>=0.2.0
```
The `algorithms_globals` was overlooked in the first release when it was moved out of Qiskit. This was added in 0.2.0 so this update needs that as a min version.